packages/libs/deer/json/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "deer-json"
+version = "0.0.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+error-stack = "0.2.4"
+
+serde_json = { version = "1.0.89", default_features = false, features = ['alloc', 'raw_value'] }
```

Comment:
It seems to work without this feature. Any specific reason why we want that feature anyway?
```suggestion
serde_json = { version = "1.0.89", default_features = false, features = ['raw_value'] }
```

Reply:
I think we can remove `raw_value` (this is from a previous attempt of mine), but `alloc` is required for `serde_json::Value` AFAIK.

Reply:
I just checked the source code, `serde_json` requires `alloc` (regardless if enabled or not), so in theory, we _could_ remove the feature. It just enables `serde/alloc`

Reply:
I don't see a reason then to have it explicit, I'd remove it.

Reply:
for me `serde_json` fails with `alloc` disabled:

<img width="843" alt="image" src="https://user-images.githubusercontent.com/7252775/206862703-db7209a5-8a67-4181-83f9-abe461117e2a.png">